java-engine/src/main/java/io/getunleash/engine/CustomStrategiesEvaluator.java

```diff
@@ -65,14 +73,20 @@ private List<MappedStrategy> getFeatureStrategies(FeatureDefinition feature) {
             mappedStrategies.add(
                     new MappedStrategy("customStrategy" + (index++), impl, strategyDefinition));
         }
+        if (fallbackStrategy != null) {
+            mappedStrategies.add(
+                    new MappedStrategy(
+                            "customStrategy" + index,
+                            fallbackStrategy,
+                            new StrategyDefinition("fallback", Collections.emptyMap())));
```

Comment:
The name doesn't really matter, this strategy has to be available for all feature toggles